src/sections/create-collection/collection-form/CollectionForm.tsx

```diff
@@ -30,7 +37,42 @@ export type CollectionFormData = {
   type: CollectionType | ''
   description: string
   contacts: { value: string }[]
+  [USE_FIELDS_FROM_PARENT]: boolean
+  [METADATA_BLOCKS_NAMES_GROUPER]: CollectionFormMetadataBlocks
+  [INPUT_LEVELS_GROUPER]: FormattedCollectionInputLevels
 }
+export type CollectionFormMetadataBlock = Exclude<
+  MetadataBlockName,
+  MetadataBlockName.CODE_META | MetadataBlockName.COMPUTATIONAL_WORKFLOW
```

Comment:
Why do we have these names in MetadataBlockName without using them? Will they be used in the future?

Reply:
This is the same thing about dynamic or not dynamic metadata blocks, will change also later, this is to narrow MetadataBlockName to all enums except those two.